File: FragmentTabLayout/app/src/main/java/com/arf/fragmenttablayout/MainActivity.java
package com.arf.fragmenttablayout;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    TabsLayout tabsLayout;
    ViewPager viewPager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tabsLayout  =   findViewById(R.id.tabLayout);
        viewPager   =   findViewById(R.id.viewPager);

        ArrayList<Fragmnet> fragmnets = new ArrayList<>();
        fragmnets.add(new ChatFragment());
        fragmnets.add(new StatusFragment());
        fragmnets.add(new CallFragment());

        ArrayList<String> judulFragmnet = new MyViewPagerAdapter(getSupportFragmentManager(), Fragment, judulFragment);

    }
}

File: ListView/app/src/main/java/com/elapp/listview/MainActivity.kt
package com.elapp.listview

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.AdapterView
import android.widget.ArrayAdapter
import android.widget.ListView
import android.widget.Toast
import androidx.databinding.DataBindingUtil
import com.elapp.listview.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var adapter: ArrayAdapter<CharSequence>

    private var _activityMainBinding: ActivityMainBinding? = null
    private val binding get() = _activityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        _activityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main)

        binding?.btnListView?.setOnClickListener {
            startActivity(Intent(this@MainActivity, ListViewActivity::class.java))
        }

        binding?.btnRecyclerView?.setOnClickListener {
            startActivity(Intent(this@MainActivity, RecyclerViewActivity::class.java))
        }

    }

}